pkg/server/auth.go

```diff
@@ -0,0 +1,154 @@
+package server
+
+import (
+	"context"
+	"log"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/zostay/hermione/pkg/auth"
+	"github.com/zostay/hermione/pkg/task"
+)
+
+type Authorizer struct {
+	mgr *task.Manager
```

Comment:
nit: spell out attributes?

```suggestion
	manager *task.Manager
```